html/themes/custom/iasc8/iasc8.info.yml

```diff
@@ -13,3 +13,5 @@ regions:
   sidebar_first: First sidebar
   sidebar_second: Second sidebar
   footer: Footer
+
+core_version_requirement: 10
```

Comment:
Got it working locally and found this error message during `drush cr`
`The 'core_version_requirement' constraint (10) requires the 'core' key not   
  be set in themes/custom/iasc8/iasc8.info.yml ` - we should delete line 4 above.